### PR TITLE
include version numbers involved with warn_for_outdated_bundler_version

### PR DESCRIPTION
**Pull request by jrochkind**
Can make it easier to understand what's going on.

### CODE REVIEW

**Comment by segiddins** [comment]
Thanks! @homu r+

**Comment by jrochkind** [comment]
sorry fixed the output to have a #to_s, I wasn't sure that `#{}` on the Gem::Version object alone does the right thing, but to_s does.

File: lib/bundler/lockfile_parser.rb [hunk 1]
```diff
@@ -109,8 +109,8 @@ def warn_for_outdated_bundler_version
         raise LockfileError, "You must use Bundler #{bundler_version.segments.first} or greater with this lockfile."
       when 0
         if current_version < bundler_version
-          Bundler.ui.warn "Warning: the running version of Bundler is older " \
-               "than the version that created the lockfile. We suggest you " \
+          Bundler.ui.warn "Warning: the running version of Bundler (#{current_version.to_s}) is older " \
```
Comment (segiddins) [inline lib/bundler/lockfile_parser.rb]: the `to_s` is implied in string interpolation, so it's redundant here

**Comment by indirect** [comment]
@jrochkind using `#{}` actually calls `to_s` on the object! so your original code was good. :)

**Comment by jrochkind** [comment]
oh, it doesn't call `inspect`? Okay, I'll change it back.

**Comment by segiddins** [comment]
👍 with updated tests, including ./spec/commands/check_spec.rb:331 && ./spec/lock/lockfile_spec.rb:154

**Comment by indirect** [comment]
looks like it's failing on Ruby 2.0+ right now... but you're close! 😀

**Comment by jrochkind** [comment]
I _think_ it's failing just on a rubocop style warning?  Do you have CI set up to fail on rubucop style warnings... but only on ruby 2.0+?  This is the extent of the travis test output:

~~~
spec/lock/lockfile_spec.rb:179:23: C: Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
    warning_message = 'the running version of Bundler (9999999.0.0) is older than the version that created the lockfile (9999999.1.0)'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
302 files inspected, 1 offense detected
RuboCop failed!
~~~

I'll try fixing the rubocop warning and see if travis passes...

**Comment by segiddins** [comment]
Yes, rubocop has to pass on CI, but we can't run it on 1.8.7 since it's incompatible

**Comment by jrochkind** [comment]
Okay, I've never had as much trouble getting something to pass CI as I have here!

Currently it's failing only on 1.8.7, with a single failure. I don't understand the failure, and I don't _think_ it's related to my code (but could be wrong?), appreciate any tips. 

~~~
  1) Bundler.with_env helpers Bundler.clean_exec runs exec inside with_clean_env
     Failure/Error: expect($?.exitstatus).to eq(42)
     
       expected: 42
            got: nil
     
       (compared using ==)
     # ./spec/runtime/with_clean_env_spec.rb:132
~~~

**Comment by segiddins** [comment]
@jrochkind we're dealing with intermittent CI failures at the moment, don't worry about it

**Comment by segiddins** [comment]
@homu r+

**Pull request merged**